.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Any reason why this was moved out of the package.json? I don't mind, I'm just curious :)

Reply:
There was a `.eslintrc.js` in the `tests` forlder, I thought I would merge both for clarity (as it is not obvious that a `.eslintrc` file and a `eslintConfig` object in the package.json are exactly the same thing). And also to lighten the package.json and separate concern, but I don't mind either putting it back :) 

Your comment made me realise I didn't remove the second .eslintrc, doing it now!